index.html

```diff
@@ -180,4 +184,4 @@ <h4 class="my-email">jose.sozasobalvarro@gmail.com</h4>
         </div>
```

Comment:
Please follow the Figma template for styling the mail address. Font size and family don't match. Kindly check the differences below and fix it. 👨‍🎨
<img width="424" alt="Screenshot 2023-01-24 at 11 22 03 PM" src="https://user-images.githubusercontent.com/25887253/214402243-d001a4a0-91fc-40a2-8bd9-0e04d9394156.png">

<img width="804" alt="Screenshot 2023-01-24 at 11 21 17 PM" src="https://user-images.githubusercontent.com/25887253/214402237-02fa9638-0919-4d30-bb5b-6c0a86a29b8c.png">